cmds/init/init.go

```diff
@@ -107,6 +107,9 @@ func main() {
 		}
 	}
 
+	// Start background build.
+	go startBgBuild()
```

Comment:
Here is where it would not be hard to figure out if we really want to do this. I think in the general case we probably want to, but ...